### PR TITLE
Card 150: double size of 'get directions' message

### PR DESCRIPTION
**Pull request by bvibber**
As requested here: https://mingle.corp.wikimedia.org/projects/wlm_android_app/cards/150

Feels a bit sketchy to me though; do we want this or do we want to improve the size of the text all around here?

### CODE REVIEW

File: assets/www/app.css [hunk 1]
```diff
@@ -383,6 +383,10 @@ button.myLocation.loading {
 	color: #333;
 }
 
+.monument-directions {
+	font-size: 200%;
```
Comment (jdlrobson) [inline assets/www/app.css:387]: mmm.. something smells wrong.  As a rule I always ensure font sizes are always no bigger than the parent element.

Maybe we need to reassess our font-sizes elsewhere.

We seem to be using a mixture of pixels, ems and %s - we should probably go with font-size: 100% on the body tag and then define all other fonts in ems. Thoughts?

Should also probably adjust #monument-detail .monument-sub-details font size instead...
Thoughts?

Comment (bvibber) [inline assets/www/app.css:387]: I think I'd rather the entire details section be in larger type; it's a bit small and anything tappable is a tough target right now...

Comment (jdlrobson) [inline assets/www/app.css:387]: Agreed

**Comment by awjrichards-zz** [comment]
I went ahead and merged this pull request as it satisfies the acceptance criteria for the card. You guys should go ahead and bring your concerns up with Phil and see about getting a new card introduced for increasing the size of the text in the details section.

**Pull request merged**